Mathlib/Geometry/Manifold/Diffeomorph.lean

```diff
@@ -530,14 +529,6 @@ theorem contMDiff_transContinuousLinearEquiv_right {f : M' → M} :
     ContMDiff I' (I.transContinuousLinearEquiv e) n f ↔ ContMDiff I' I n f :=
   (toTransContinuousLinearEquiv I M e).contMDiff_diffeomorph_comp_iff le_rfl
 
-@[deprecated (since := "2025-06-12")]
-alias _root_.Diffeomorph.contMDiff_transDiffeomorph_right :=
-contMDiff_transContinuousLinearEquiv_right
```

Comment:
Okay this one is too early.
```suggestion
@[deprecated (since := "2025-06-12")]
alias _root_.Diffeomorph.contMDiff_transDiffeomorph_right :=
contMDiff_transContinuousLinearEquiv_right

```